src/axom/quest/readers/ProEReader.hpp

```diff
@@ -73,6 +107,28 @@ class ProEReader
    */
   virtual int read();
 
+  /// \name Set tetrahedron predicate to read mesh subset
+  /// @{
+  /*!
+   * The reader calls the \a TetPred \a p after reading each tet,
+   * retaining tets where p returns true and discarding the rest.
+   * If a TetPred is not set, the reader retains all tets.
+   */
+  virtual void setTetPred(const TetPred& p) { m_tetPredicate = p; }
+
+  /*!
+   * Convenience function to set a \a TetPred from a bounding box.
+   * 
+   * \param box The TetPred will keep all tets with all four nodes in the
+   *        box.  If the box is invalid, no TetPred is constructed, so all
+   *        tets are retained.
```

Comment:
Please update this -- it no longer seems to match the functionality.

Perhaps something like the following:

```suggestion
   * \param box The TetPred will be defined in terms of the provided bounding box.  
   *             If the box is invalid, no TetPred is constructed and all tets are retained.
```

Reply:
Updated.